engine/polyglot-api/src/main/scala/org/enso/polyglot/runtime/Runtime.scala

```diff
@@ -193,6 +193,10 @@ object Runtime {
         value = classOf[Api.SuggestionsDatabaseModuleUpdateNotification],
         name  = "suggestionsDatabaseModuleUpdateNotification"
       ),
+      new JsonSubTypes.Type(
+        value = classOf[Api.AnalyzeModuleInScopeJobFinished],
+        name  = "analyzeModuleInScopeJobFinished"
```

Comment:
This defines new message in the protocol. Shall not that [be documented](https://github.com/enso-org/enso/blob/develop/docs/language-server/protocol-language-server.md)?